packages/shared/util.ts

```diff
@@ -6,6 +6,10 @@ export function truncateText(text: string, max_len: number) {
   }
 }
 
+/**
+ * @returns Whether the `url` is a contact or a group invite link.
+ * (https://github.com/deltachat/invite/blob/1b5b2e2a9bfcaf87c4f44929fb7734819d1d9679/README.md?plain=1#L6-L7)
```

Comment:
maybe just like to the general page and additionally the specification for these links:

- https://github.com/deltachat/invite
- https://github.com/deltachat/interface/blob/main/uri-schemes.md#invite-links-httpsideltachat-

Reply:
```suggestion
 * - https://github.com/deltachat/invite
 * - https://github.com/deltachat/interface/blob/main/uri-schemes.md#invite-links-httpsideltachat-
```